docs/_getting_started/jupyter.md

```diff
@@ -75,29 +74,42 @@ There are a few preliminary steps that need to be taken before SAS can be used o
 ### I don't see Julia on my JupyterHub!
 ---------------------------------------
 
-See our [FAQ page](/faqs/installJuliaOnJupyter.html) on how to install it!
+See our [User Guide](/_user_guide/best_practices_julia_jupyter/) on how to install it!
 
 
 ### Consoles and Terminal
 -------------------------
-![](/images/jupyter_consoles.png "Consoles")
+![](/assets/images/jupyter_consoles.png "Consoles")
 
 You can also launch interactive consoles from JupyterHub.  These will behave very similar to the versions on the Yen servers.
 
-![](/images/jupyter_terminal.png "Terminal")
+![](/assets/images/jupyter_terminal.png "Terminal")
 
 You can also launch a bash terminal from JupyterHub.  This provides access to commands you would normally run on the Yens, but through the web browser.
 
 ### File Upload and Download
 ----------------------------
-![](/images/jupyter_upload.png "File Upload")
+![](/assets/images/jupyter_upload.png "File Upload")
 
 One very useful feature of JupyterHub is the ability to upload and download files from ZFS/AFS.  First, make sure you are in the proper directory.  Then, to upload, click the up arrow on the top left of your screen to select a file.
 
-![](/images/jupyter_download.png "File Download")
+![](/assets/images/jupyter_download.png "File Download")
 
 To download, right click the file you would like, and click "Download".
 
+
+### Text File Editor
+-------------------------
+![](/assets/images/editor.png)
+
+Finally, you can also edit text files like R scripts directly on JupyterHub. Clicking on Text File icon will open a new file that you can edit. Similarly, clicking on Python File will create an empty `.py` file and clicking on R File will create an empty `.r` file.
+You can also navigate to a directory that has the scripts you want to edit and double click on the script name to open it up in the Text Editor.
+
+For example, try to navigate to `intro_yens_sep_2023` folder in file brower first then double click on `investment-npv-parallel.R` file to open it in the text editor:
+![](/images/edit-r-script.png)
+
+
+
 ## Technical Details
 
 ### File Access and Storage
```

Comment:
we should remove access to afs here, and also provide correct installation instructions for pacakges.  Python, use venv.  R, use R in a ssh session.

Reply:
done